distribution/options.html

```diff
@@ -28,6 +28,13 @@
 
 	<hr>
 
+	<p>
+		<label>
+			<strong>Action button URL</strong><br>
```

Comment:
I think we need a better label here, "action" is a webext term. Maybe:

``` html
<strong>Refined GitHub icon link</strong>
```

Reply:
How about: "Open this URL with the toolbar button" ?

Reply:
"Extension icon link" would be enough.

Reply:
I think "Extension" makes it a little less clear. Florent’s suggestion could work well as an additional label, e.g.:


```html
<strong>Refined GitHub icon link</strong> this page will open when clicking the toolbar icon
```

Reply:
@cheap-glitch Just pick one that makes sense to you and merge as you see fit, we don't need to spend too much time perfecting this 😁